misc/xml-test-files/languages.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<debate-format schema-version="2.2">
+  <name xml:lang="en">Weird languages</name>
+  <name xml:lang="es">Idiomas extraños</name>
+  <name xml:lang="fr">Langues étranges</name>
+  <name xml:lang="zh">奇怪的语言</name>
```

Comment:
If you want more "weird"*, I recommend Arabic or Hebrew, i.e. a right-to-left language.

*) Well, for us Western people anyway ;)

Reply:
Haha, done! Though since it's not the whole interface in RTL I think it doesn't make much difference, for now.

Admittedly "Weird languages" is a bit of a misnomer for this file—it's the file that's weird, not the languages 😉